django/app_tutorial/models.py

```diff
@@ -4,9 +4,6 @@
 from django.contrib.auth.models import User
 
 
-# TODO (Feedback): Sort classes alphabetically or logically?
```

Comment:
I ran into the same issue before. An incorrect order of db table class definitions in Django's model.py caused an error. In case you've also wondered why this happens, here is an explanation I came up with.  

The behavior is not Django-specific but Python related. Class attributes are interpreted by CPython right away and their execution is not deferred until the Class is called for object instantiation. In our example, the reference for `FieldType` is used in line 24. However, `FieldType` is not known to the module's namespace until line 27. The correct exception should be a `NameError`:

```python

class FirstClass:
    pass

class SecondClass:
    field_a = FirstClass # works as expected
    field_b = ThirdClass # raises NameError exception

class ThirdClass:
    pass

```

In contrast, functions/methods are only executed upon being called. Therefore, they are allowed to use variable references which are not yet available to the module's namespace:

```python

class FirstClass:
    pass

class SecondClass:
    def __init__(self):
        self.field_a = FirstClass
        self.field_b = ThirdClass

class ThirdClass:
    pass


if __name__ == "__main__":
    dummy = SecondClass()
```